corehq/apps/prototype/static/prototype/js/webpack/bootstrap3_amd.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+hqDefine("prototype/js/webpack/bootstrap3_amd",[
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    "commcarehq_b3",  // IMPORTANT :: this has to be included with any Bootstrap 3 entry point
```

Comment:
"Bootstrap 3 entry point" => Does this mean a javascript module that explicitly depends on bootstrap 3 javascript? Or all javascript modules used on pages that use bootstrap 3?

Reply:
i think i'm having trouble understanding the difference here?

Reply:
Explicitly depends on bootstrap 3: [data dictionary](https://github.com/dimagi/commcare-hq/blob/7d461505b0695e457b73bca737b9a8fd3f933536/corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js#L7)

Entry point for a B3 page that does not explicitly depend on B3 js: [case details report](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/static/reports/js/bootstrap3/case_details.js)